firmware/library/L4_Application/Temperature_Sensor.hpp

```diff
@@ -0,0 +1,21 @@
+/* Interface used to obtain the temperature of the system.*/
+#pragma once
+
+class TemperatureInterface
+
+{
+ public:
+  virtual bool Init() = 0;
+  // Initialize Drivers
```

Comment:
Um, comments are usually, and most intuitively placed above the line it talks about. But regardless, the comment is superfluous. The name of the method already tells you what it does.